components/map/MapScreen.tsx

```diff
@@ -33,8 +33,8 @@ const MapScreen = () => {
   return (
     <View className="flex-1 items-stretch">
       <Map onBottomSheetContentChange={handleBottomSheetChange} />
-      {false && (
-        <BottomSheet>
+      {bottomSheetContent && (
+        <BottomSheet snapPoints={['20%', '80%']}>
```

Comment:
In docs, they use `useMemo` on snapPoints. I don't know yet if it's needed, but just to let you know they use it :D